### PR TITLE
Fix issue in docs: typo in dockerfile name, and typo in url.

### PR DESCRIPTION
**Pull request by johnzheng1975**
Signed-off-by: John Zheng <john.zheng@hp.com>

"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/modelserving/v1beta1/transformer/torchserve_image_transformer/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ class ImageTransformer(kserve.Model):
         return infer_request
 ```
 
-Please see the code example [here](https://github.com/kserve/kserve/tree/release-0.11/python/custom_transformer).
```
Comment (spolti) [inline docs/modelserving/v1beta1/transformer/torchserve_image_transformer/README.md:73]: guess we still need to point to the latest release, 0.14 or 0.13.
cc @yuzisun

Comment (yuzisun) [reply docs/modelserving/v1beta1/transformer/torchserve_image_transformer/README.md:73]: I wonder if there is way to use a release branch variable here and then resolve.

**Comment by johnzheng1975** [comment]
resolved, pls review again. thanks.

**Comment by johnzheng1975** [comment]
@yuzisun  @spolti could you help to review,  thanks.

**Comment by johnzheng1975** [comment]
You know, the doc cannot be run successfully without this PR. This is why should merge this PR.
About how to improve this -- need not change this  for each release, this is another goal/PR handle later.

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**